cluster/addons/cluster-monitoring/google/heapster-service.json

```diff
@@ -0,0 +1,23 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "name": "heapster",
+      "kubernetes.io/cluster-service": "true"
+    },
+    "name": "monitoring-heapster"
+  },
+  "spec": {
+    "ports": [
+      {
+        "port": 80,
+        "targetPort": 8082
```

Comment:
Good point. The heapster service is not needed, I will remove it from both influxdb and google.